src/Pos/DB/Block/Block.hs

```diff
@@ -0,0 +1,302 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+
+-- | Interface to Blocks DB.
```

Comment:
I'd like this comment to say a) how we store blocks, and b) why we're not storing them in rocksdb